pkg/model/token.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+import "fmt"
+
+type Token string
+
+func (t *Token) Bearer() string {
+	return fmt.Sprintf("Bearer %s", t)
+}
+
+func (t *Token) String() string {
+	return string(*t)
+}
```

Comment:
This is a hella minor nit but just for educational purposes: It's likely better to make this take an instance receiver instead of a pointer receiver. One less dereference to do in the `String()` function! It also indicates to the consumer that these are read-only functions.